lib/network.js

```diff
@@ -51,63 +58,51 @@ class Network {
             }
         });
 
-        this.requests = (callback) => {
+        this.requests = () => {
 
-            callback(null, this._requests);
+            return this._requests;
         };
 
-        this.concurrents = (callback) => {
```

Comment:
We're dropping this because hapi no longer supports multiple connections, correct?

Reply:
Correct!